src/feature-viewer/index.ts

```diff
@@ -25,48 +31,42 @@ export class FeatureViewerComponent extends ActionLitElement {
 
   override render() {
     if (this.error) {
-      // TODO: Is this not a feature with a error modifier? With a classname `feature feature--error`?
-      return html`
-        <div class="feature__error">
-          ${this.error.message}
-        </div>
-      `;
+      return html`<bdo-alert type="error">${this.error.message}</bdo-alert>`;
     }
 
     if (!this.feature) {
       return null;
     }
 
     return html`
-      <div class="${this.getFeatureClass()}">
-        <div class="feature__header">
-          <h2 class="feature__title">Feature: ${this.feature.name}</h2>
-          ${this.feature.tags?.length ? html`
-            <p class="feature__tags">
-              ${this.feature.tags.map((tag) => html`<bdo-badge type="tag">${tag}</bdo-badge>`)}
-            </p>`: null}
-          ${this.feature.description ? html`
-            <p class="feature__description">
-              ${this.feature.description}
-            </p>`: null}
-          ${this.feature.resultFile ? html`
-            <a
-              href="${this.feature.resultFile}"
-              class="feature__result-link"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              View Test Results
-            </a>`: null}
-        </div>
-        <div class="feature__content">
-          ${this.feature.background ? html`<feature-background .background=${this.feature.background}></feature-background>`: null}
-          ${this.feature.scenarios.map((scenario) =>
-          "examples" in scenario
-            ? html`<feature-scenario-outline .outline=${scenario}></feature-scenario-outline>`
-            : html`<feature-scenario .scenario=${scenario}></feature-scenario>`
-          )}
-        </div>
+      <div class="feature__header">
```

Comment:
@stiekstra zou dit niet een `bdo-heading-container`moeten zijn? Het is toch dezelfde functionaliteit?